test_data/config.toml

```diff
@@ -0,0 +1,206 @@
+[vars]
+FLOX_FEATURES_USE_CATALOG = "true"
+
+[resolve.empty]
+# Not actually a resolve response, just have to put it somewhere
+skip_if_output_exists = "empty.json"
+cmd = '''
+    echo "[]" > empty.json
+'''
+post_cmd = '''
+    main_output_dir="$(dirname $(dirname $RESPONSE_FILE))"
+    mv empty.json "$main_output_dir/empty.json"
```

Comment:
note nonblocking: it's hard for me to understand what's actually going on here

Reply:
This is my hack for putting multiple output files into a directory named after the job rather than generating just a single JSON file. Docs could be better but that's an armadillo for now.